PowerShellGet/PSModule.psm1

```diff
@@ -4906,6 +4907,15 @@ function Test-ScriptFileInfo
             return
         }
 
+        if ($PSScriptInfo.Version -match '-')

```

Comment:
I'm assuming version validation is being done elsewhere as part of the initial implementation of Test-ScriptFileInfo.  My work is to specifically add prerelease validation, not overall version validation.